src/pheromones/include/pheromones/HiveConnectNetworkConfigSetRequestDTO.h

```diff
@@ -0,0 +1,32 @@
+#ifndef PROPOLIS_HIVECONNECTNETWORKCONFIGSETREQUESTDTO_H
+#define PROPOLIS_HIVECONNECTNETWORKCONFIGSETREQUESTDTO_H
+
+#include "HiveConnectMeshEnableDTO.h"
+#include "HiveConnectNetworkAccessDTO.h"
+#include "HiveConnectRootNodeDTO.h"
+#include "message.pb.h"
+#include <optional>
+
+class HiveConnectNetworkConfigSetRequestDTO {
+  public:
+    HiveConnectNetworkConfigSetRequestDTO();
```

Comment:
Is the default constructor used anywhere?

Reply:
It will be used to have an object to which you can then set the settings for the network easier than having to provide everything at construction.

Reply:
Generally it's better to make sure your object is valid at construction. always favor RAII when possible.

Reply:
Generally it's recommended to use RAII,  here it's fine not to since you use optional fields. Though you could create a constructor with default arguments values too.